File: 2272.java
import java.util.*;
public class Main {

  public static int score(double x, double y) {
    double r = x * x + y * y;
    if (r <= 9.)
      return 100;
    if (r <= 36.)
      return 80;
    if (r <= 81.)
      return 60;
    if (r <= 144.)
      return 40;
    if (r <= 225.)
      return 20;
    return 0;
  }
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    double x, y;
    int sum1, sum2;
    while (true) {
      sum1 = 0;
      sum2 = 0;
      x = sc.nextDouble();
      y = sc.nextDouble();
      if (x == -100)
        break;
      sum1 += score(x, y);
      for (int i = 1; i <= 2; i++) {
        x = sc.nextDouble();
        y = sc.nextDouble();
        sum1 += score(x, y);
      }
      for (int i = 1; i <= 3; i++) {
        x = sc.nextDouble();
        y = sc.nextDouble();
        sum2 += score(x, y);
      }
      if (sum1 > sum2)
        System.out.println("SCORE: " + sum1 + " to " + sum2 +
                           ", PLAYER 1 WINS.");
      else if (sum1 == sum2)
        System.out.println("SCORE: " + sum1 + " to " + sum2 + ", TIE.");
      else
        System.out.println("SCORE: " + sum1 + " to " + sum2 +
                           ", PLAYER 2 WINS.");
    }
  }
}

File: 2272.cc
#include <iostream>
#include <cstdio>
#include <cstdlib>
#include <cstring>
#include <cmath>
using namespace std;

#define maxn 10
#define eps 10E-9

struct Point {
  double x, y;
} f[maxn], g[maxn];

int a, b;

int dblcmp(double a, double b) {
  if (abs(a - b) < eps) return 0;
  if (a > b) return 1;
  return -1;
}

void input() {
  for (int i = 0; i < 3; i++) scanf("%lf%lf", &f[i].x, &f[i].y);
  for (int i = 0; i < 3; i++) scanf("%lf%lf", &g[i].x, &g[i].y);
}

double dist(Point &a) { return a.x * a.x + a.y * a.y; }

double score(double a) {
  if (dblcmp(a, 9) <= 0) return 100;
  if (dblcmp(a, 36) <= 0) return 80;
  if (dblcmp(a, 81) <= 0) return 60;
  if (dblcmp(a, 144) <= 0) return 40;
  if (dblcmp(a, 225) <= 0) return 20;
  return 0;
}

void work() {
  a = b = 0;
  for (int i = 0; i < 3; i++) a += score(dist(f[i]));
  for (int i = 0; i < 3; i++) b += score(dist(g[i]));
  if (a == b)
    printf("SCORE: %d to %d, TIE.\n", a, b);
  else if (a < b)
    printf("SCORE: %d to %d, PLAYER 2 WINS.\n", a, b);
  else if (a > b)
    printf("SCORE: %d to %d, PLAYER 1 WINS.\n", a, b);
}

int main() {
  while (1) {
    input();
    if (dblcmp(f[0].x, -100) == 0) break;
    work();
  }
  return 0;
}
